src/test/test-parse-util.c

```diff
@@ -922,4 +922,23 @@ TEST(parse_loadavg_fixed_point) {
         assert_se(parse_loadavg_fixed_point("", &fp) == -EINVAL);
 }
 
+TEST(pcr_from_string) {
+        assert_se(pcr_number_from_string((char*)"core-firmware") == 0);
```

Comment:
Why the cast?

Reply:
> Why the cast?

I wonder what I was thinking. @bluca has guided well enough on that.